playbooks/roles/jenkins/templates/jenkins_defaults.j2

```diff
@@ -36,6 +36,9 @@ RUN_STANDALONE=true
 # log location.  this may be a syslog facility.priority
 JENKINS_LOG={{ jenkins_logs_dir }}/$NAME.log
 
+# enable access log; see https://wiki.jenkins.io/display/JENKINS/Access+Logging
+JENKINS_ACCESSLOG="--accessLoggerClassName=winstone.accesslog.SimpleAccessLogger --simpleAccessLogger.format=combined --simpleAccessLogger.file=/var/log/jenkins/access.log"
```

Comment:
Daft question but who/what is `winstone`?

Reply:
Also on those steps it says `add  $JENKINS_ACCESSLOG to JENKINS_ARGS` - guessing we don't need that here?